shm/Map.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <vector>
+
+#include "Island.hpp"
+
+constexpr int randomNumberStart = 1;
```

Comment:
It's better to have one naming concept. 
```suggestion
constexpr int minPositionXY = 1;
```